adm/style/acp_phpbb_ideas.html

```diff
@@ -28,16 +28,17 @@
 			<dd>{options.CONTENT}</dd>
 			<!-- IF options.KEY == 'ideas_poster_id' --><dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd><!-- ENDIF -->
 		</dl>
+		<!-- IF options.KEY == 'ideas_base_url' -->
```

Comment:
The buttons should just go after the end of the options, before the closing fieldset. No conditional needed.